### PR TITLE
Fixes ubuntu desktop image error of package not found

### LINKED ISSUE #473: Failed to setup node following the quickstart guide
**Describe the bug**
On a fresh install ubuntu 20.04 image, the node setup script failed to run due to missing required dependencies.

**To Reproduce**
On a fresh install ubuntu 20.04 image, run the following commands to configure the node following the [quickstart guide:](https://github.com/ease-lab/vhive/blob/main/docs/quickstart_guide.md#1-setup-all-nodes)

```
sudo apt update
sudo apt install git

git clone --depth=1 https://github.com/ease-lab/vhive.git
cd vhive
mkdir -p /tmp/vhive-logs
./scripts/cloudlab/setup_node.sh > >(tee -a /tmp/vhive-logs/setup_node.stdout) 2> >(tee -a /tmp/vhive-logs/setup_node.stderr >&2)
```
Then the script `setup_node.sh` will drop several errors due to missing required dependencies. Please check Section [Logs](#logs) for detail information.

**Expected behavior**
Configure the node successfully

**Logs**
Terminal log:

```
ubuntu@ubuntu-KVM:~/Downloads/vhive$ ./scripts/cloudlab/setup_node.sh > >(tee -a /tmp/vhive-logs/setup_node.stdout) 2> >(tee -a /tmp/vhive-logs/setup_node.stderr >&2)
APT::Periodic::Update-Package-Lists "0";
APT::Periodic::Download-Upgradeable-Packages "0";
APT::Periodic::AutocleanInterval "0";
APT::Periodic::Unattended-Upgrade "0";
Created symlink /etc/systemd/system/apt-daily.service → /dev/null.
Created symlink /etc/systemd/system/apt-daily-upgrade.service → /dev/null.
/home/ubuntu/Downloads/vhive/scripts/setup_system.sh: line 28: curl: command not found
Warning: apt-key output should not be parsed (stdout is not a terminal)
gpg: no valid OpenPGP data found.
E: The repository 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04  Release' does not have a Release file.
E: Unable to locate package skopeo
sysctl: setting key "net.ipv4.conf.all.promote_secondaries": Invalid argument
sudo: route: command not found
sudo: nft: command not found
sudo: nft: command not found
sudo: nft: command not found
sudo: nft: command not found
sudo: nft: command not found
sudo: nft: command not found
```

### PR DESCRIPTION
**Pull request by harshitgarg22**
## Summary

Fixes #473 .

## Implementation Notes :hammer_and_pick:

* Adds universe repository in ubuntu
* Adds ca-certificates package to pull from opensuse repo

## External Dependencies :four_leaf_clover:

* N/A

## Breaking API Changes :warning:

* N/A

*Simply specify none (N/A) if not applicable.*

### CODE REVIEW

**Comment by harshitgarg22** [comment]
@dhuang-esl can you please confirm if this works?

File: scripts/setup_system.sh [hunk 1]
```diff
@@ -24,6 +24,10 @@
 
 # Add skopeo sources
  . /etc/os-release
+sudo apt-get -y install curl >> /dev/null
```
Comment (ustiugov) [inline scripts/setup_system.sh:27]: Please combine lines 27 and 29

**Review by ustiugov — CHANGES_REQUESTED** [review]
Please remove the merge commit, and rebase your branch on top of main. Please read the docs/contributing*.md for the details how to do that.

**Comment by harshitgarg22** [comment]
> Please remove the merge commit, and rebase your branch on top of main. Please read the docs/contributing*.md for the details how to do that.

Please check if this is fine. Thanks for the heads up.

**Comment by ustiugov** [comment]
GitHub still shows 2 commits in this PR

File: scripts/setup_system.sh [hunk 2]
```diff
@@ -24,6 +24,9 @@
 
 # Add skopeo sources
```
Comment (ustiugov) [inline scripts/setup_system.sh:25]: This comment is for the curl command, please move it down

**Review by ustiugov — CHANGES_REQUESTED** [review]
- [ ] Need to have 1 commit, please use 'git rebase -i' to get rid of the merge commit. 
- [ ] Move the comment as specified

**Comment by harshitgarg22** [comment]
> * [ ]  Need to have 1 commit, please use 'git rebase -i' to get rid of the merge commit.
> 
>     * [ ]  Move the comment as specified

The two commits were for the original change and the combining of lines. I've squashed all changes into one now. Moved the comment above the curl command.

**Review by ustiugov — APPROVED** [review]

**Pull request merged**